cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.md

```diff
@@ -76,7 +79,7 @@ The DELPHI stack is mostly written in Fortran, with some bits written in C. Only
 
 The collaboration main contact for data preservation is the mailing list DELPHI-data-preservation-board@cern.ch. Support can only be given on a best effort basis. Suggestions and feedback is of course welcome!
 
-# Examples
+# <a name="examples"> Examples
```

Comment:
Cosmetics: the Documentation and Examples are sibling section (as per table of contents), but former uses two leading hash marks and latter one leading hash mark. We may want to standardise the section level depth:

```
 grep '^#' cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.md
# Table of contents
## <a name="overview"> Overview
### Before you start ...
## <a name="access"> Accessing the software stack
### Docker
### CVMFS
#### Requirements
#### Initialising the environment:
## <a name="documentation"> Documentation
## Source code
## Compilers
## Getting help
# <a name="examples"> Examples
## About simulation
## Creating Monte Carlo samples interactively
### Using internal generators
### Using external generators
## Running an analysis job on the result
## Using nicknames
## Raw data access
## Event visualisation:
## More examples
```

Reply:
Cosmetics: The titles "Initialising the environment" and "Event visualisation" should be without tailing colon punctuation sign.

Reply:
punctuation fixed